metalog.js

```diff
@@ -294,7 +294,7 @@ class Logger extends events.EventEmitter {
           resolve();
           fs.stat(fileName, (err, stats) => {
             if (!err && stats.size === 0) {
-              fsp.unlink(this.file);
+              fsp.unlink(fileName).catch(reject);
```

Comment:
This will not work, the promise is already resolved 3 lines above.
The only result from this is an error from node.js  https://nodejs.org/api/process.html#event-multipleresolves

Reply:
This should either be an "empty"/do-nothing catch or move the resolve to the unlink as well.

Reply:
@lundibundi Thanks, corrected